### DISCUSSION TITLE
sfdx force:package:install --package to accept LATEST placeholder

### DISCUSSION THREAD

**Discussion by baksale** [forcedotcom/cli#2361] [Ideas]
**Is your feature request related to a problem? Please describe.**
In many CI/CD activities it is required to install latest package version (e.g. in java world there is a -SNAPSHOT placeholder which guarantees latest version).
We'd like to have the same with dx packages.
https://salesforce.stackexchange.com/questions/273531/sfdx-command-to-find-the-latest-package-version

**Describe the solution you'd like**
Given I have a package with `alias "My Model"`
When I run installation command like this 
`sfdx force:package:install --package "My Model@LATEST"`
Then sfdx finds latest created version for the package
And sfdx installs the found version

**Describe alternatives you've considered**
Alternatively, we might implement a plugin to find a version, but doesn't make sense to spread package versions-related commands across multiple plugins
As of now I'm using bash script in my CI/CD

**Comment by ghost** [comment]
This would be very useful for our team and would be simpler and cleaner than constructing a bash script to accomplish the same. Thanks!

**Comment by hamedizadpanah-ibm** [comment]
would be great to have this as some packages on AppExchange deprecate old versions which cause automated deployment failure! 

Ideally, this should work with (app listing Id instead of version) + @latest to install the latest package
`sfdx force:package:install --package a0N3A00000EJH4yUAH@latest`

**Comment by bobalicious** [comment]
Came here to say this.  Was very surprised that this didn't already form part of the sfdx commands, and would go so far as to say that this should be re-classified as a bug.

I.E. given a package with the name `packageName`, I would expect the following to work and install the latest version of the package.:

`sfdx force:package:install --package packageName`

**Comment by cinicola** [comment]
Very important gap to be resolved! I have a project with many packages, with their dependencies. I want simply to do a install for each package, in their latest version. Please, the simple `--latest` can save us.
